src/components/shared/TreeView.js

```diff
@@ -187,6 +190,7 @@ type TreeViewRowScrolledColumnsProps<DisplayData: Object> = {|
   +indentWidth: CssPixels,
 |};
 
+// This is a false-positive, as it's used as a generic trait bounds.
 class TreeViewRowScrolledColumns<
   DisplayData: Object
```

Comment:
```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/components/app/ZipFileViewer.js:279:29

Cannot create TreeView element because:
 • inexact object type [1] is incompatible with exact ZipDisplayData [2] in property mainColumn.component.displayData.
 • inexact object type [3] is incompatible with exact ZipDisplayData [2] in property mainColumn.component.displayData.
 • property name is missing in object type [1] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.
 • property name is missing in object type [3] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.
 • property url is missing in object type [1] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.
 • property url is missing in object type [3] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.
 • property zipTableIndex is missing in object type [1] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.
 • property zipTableIndex is missing in object type [3] but exists in ZipDisplayData [2] in property
   mainColumn.component.displayData.

     src/components/app/ZipFileViewer.js
 [2]  68│   +displayData: ZipDisplayData,
        :
     272│                 <h1>Firefox Profiler</h1>
     273│                 <p>Choose a profile from this zip file</p>
     274│               </header>
     275│               <TreeView
     276│                 maxNodeDepth={zipFileMaxDepth}
     277│                 tree={zipFileTree}
     278│                 fixedColumns={this._fixedColumns}
     279│                 mainColumn={this._mainColumn}
     280│                 onSelectionChange={changeSelectedZipFile}
     281│                 onExpandedNodesChange={changeExpandedZipFile}
     282│                 selectedNodeId={selectedZipFileIndex}
     283│                 expandedNodeIds={expandedZipFileIndexes}
     284│                 ref={this._takeTreeViewRef}
     285│                 contextMenuId="unused"
     286│                 rowHeight={30}
     287│                 indentWidth={15}
     288│                 onEnterKey={this._onEnterKey}
     289│               />
     290│               <DragAndDropOverlay />
     291│             </div>
     292│           </section>

     src/components/shared/TreeView.js
 [1] 106│   +displayData: DisplayData,
        :
 [3] 174│   +displayData: DisplayData,
```

This gives an error as well. I think this has a different semantic meaning. Plus, I think the `: Object` will survive a TS migration better.